### ISSUE TITLE
Category marks video as game

### ISSUE THREAD

**Issue by GeorgeAnton1** [llakssz/CIAngel#103]
There seems to be some occurrence (possibly all) where the category says that a video is a game. 
I only tested this with Luigis mansion (eu).

**Comment by llakssz** [comment]
Yes, some titles are E3 promo videos.
In my own implementation of the json creator I ignored these but since we do not control the json file anymore, we will have to ignore them on the fly.
It is safe to ignore titles that contain " E3 " :+1:

**Comment by GeorgeAnton1** [comment]
Why don't you control the json creator?

**Comment by noirscape** [comment]
The guy behind the titlekey site is the one who manages the file. cearp
used to manage it, but the guy behind the titlekey site found out how the
format worked, and since then he is the one that manages the file because
it is automated on that site (cearp had to manually generate it each time).

**Comment by DrakiaXYZ** [comment]
As ev1l0rd said, it's mostly convenience. The titlekey database site created a JSON format almost exactly the same as ours, so it was just easier to use that than continue to create our own and find somewhere to host it/link to it.

**Comment by cerea1killer** [comment]
Exactly. And this way users don't have to find the latest wings.json
themselves. That was the big push why we changed that

**Comment by GeorgeAnton1** [comment]
Oh right, I didn't realise it was coming straight from that title key site. 
Would ignoring the videos on the fly be a possibility?

**Comment by llakssz** [comment]
Sure it's possible, easy in fact :)
We already skip some stuff, like system title, etc, this would just be another thing to skip.
Everything that contains " E3 " is just a promo video, there are some videos that don't obey to this, but this gets many of them.
'Picross e3' would not be skipped because " E3 " would not find it :+1:
